go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/Masterminds/squirrel v1.5.4
 	github.com/PaesslerAG/gval v1.2.2
 	github.com/PaesslerAG/jsonpath v0.1.1
+	github.com/apache/arrow/go/parquet v0.0.0-20211112161151-bc219186db40
```

Comment:
Please send go.mod file changes in separate commits. It improves readability and it's easier to rebase commits lated.